pkg/rtc/transport.go

```diff
@@ -617,7 +642,7 @@ func (t *PCTransport) handleConnectionFailed(forceShortConn bool) {
 		isShort, duration = t.IsShortConnection(time.Now())
 		if isShort {
 			pair, err := t.getSelectedPair()
-			t.params.Logger.Debugw("short ICE connection", "error", err, "pair", pair, "duration", duration)
+			t.params.Logger.Debugw("short ICE connection", "error", err, "pair", wrappedICECandidatePairLogger{pair}, "duration", duration)
```

Comment:
although `Debugw`, this could have logged remote IPs/port I think. So, changing it to use the wrapped logger which drops remote IP/port.